packages/atomic/src/pages/test.html

```diff
@@ -8,6 +8,11 @@
     <script type="module" src="/build/atomic.esm.js"></script>
     <script nomodule src="/build/atomic.js"></script>
     <link rel="stylesheet" href="/themes/accessible.css" />
+    <style>
```

Comment:
The ripple effect, when being active for a brief moment, cause some accessibility test to fail, putting a low opacity effect seems to fix it while not removing the effect entirely